topi/include/topi/elemwise.h

```diff
@@ -56,6 +56,72 @@ TOPI_DECLARE_UNARY_OP(round);
 TOPI_DECLARE_UNARY_OP(trunc);
 TOPI_DECLARE_UNARY_OP(abs);
 
+
+/* \brief Fast_tanh_float implementation from Eigen */
```

Comment:
Could you add a link to `https://github.com/eigenteam/eigen-git-mirror/blob/80f488a7b/Eigen/src/Core/MathFunctionsImpl.h#L18-L67`. That would help track the provenance in case their impl changes in the future.

Reply:
Added